compact_str/src/tests.rs

```diff
@@ -365,3 +366,87 @@ fn test_plus_operator() {
     assert_eq!(CompactStr::from("a") + String::from("b"), "ab");
     assert_eq!(String::from("a") + CompactStr::from("b"), "ab");
 }
+
+macro_rules! to_compact_str {
+    ( $fmt:expr $(, $args:tt)* ) => {
+        ToCompactStr::to_compact_str(
+            &core::format_args!(
+                $fmt,
+                $(
+                    $args,
+                )*
+            )
+        )
+    };
+}
```

Comment:
Is the purpose of this macro to be similar to `format!(...)`? If so, could we rename this to `format_compact!(...)`

Reply:
Done!